tests/unit_tests/test_wave_06.py

```diff
@@ -143,16 +139,15 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
+    assert result is True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_c in jesse.inventory
+    assert tai.inventory == [item_b, item_a, item_f]
+    assert jesse.inventory == [item_e, item_d, item_c]
```

Comment:
Same comment as above, since we aren't guaranteeing order of items in one's inventory after a swap, then these two assertions would fail if the items did not appear in the exact order as you've written on lines147-148. 

```py
    assert item_a in tai.inventory
    assert item_b in tai.inventory
    assert item_f in tai.inventory
    assert item_d in jesse.inventory
    assert item_e in jesse.inventory
    assert item_c in jesse.inventory